MediaBrowser.Controller/Lyrics/LyricResponse.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// LyricResponse model.
+/// </summary>
+public class LyricResponse
+{
+    /// <summary>
+    /// Gets or sets Metadata.
+    /// </summary>
+    public LyricMetadata? Metadata { get; set; }
```

Comment:
```suggestion
    public LyricMetadata Metadata { get; set; }
```

All properties of LyricMetadata are nullable so this one doesn't need to be nullable.